go.mod

```diff
@@ -17,7 +17,7 @@ require (
 	github.com/dgryski/go-farm v0.0.0-20200201041132-a6ae2369ad13 // indirect
 	github.com/enigmampc/btcutil v1.0.3-0.20200723161021-e2fb6adb2a25
 	github.com/gogo/gateway v1.1.0
-	github.com/gogo/protobuf v1.3.2
+	github.com/gogo/protobuf v1.3.3
```

Comment:
@aaronc I'm unable to install cosmos-sdk because it looks like this protobuf version doesn't actually exist in the repo. Here's my output:
```
$ go get -t github.com/cosmos/cosmos-sdk
go get: github.com/cosmos/cosmos-sdk@v0.40.0 updating to
        github.com/cosmos/cosmos-sdk@v0.42.5 requires
        github.com/gogo/protobuf@v1.3.3: reading https://gomodules.cbhq.net/github.com/gogo/protobuf/@v/v1.3.3.mod: 410 Gone
        server response: not found: github.com/gogo/protobuf@v1.3.3: invalid version: unknown revision v1.3.3
```

Should we revert to an existing version?

Reply:
The cosmos SDK uses a fork of gogo proto, you should add the line `replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1`, see the SDK's go.mod file